pkg/kubectl/cmd/proxy.go

```diff
@@ -152,7 +152,7 @@ func RunProxy(f cmdutil.Factory, out io.Writer, cmd *cobra.Command) error {
 	if err != nil {
 		glog.Fatal(err)
 	}
-	fmt.Fprintf(out, "Starting to serve on %s\n", l.Addr().String())
+	fmt.Fprintf(out, "Starting to serve on http://%s\n", l.Addr().String())
```

Comment:
But it could be `https`, right? Need to include other schema as well.

Reply:
As far as I can understand the code, the local proxy server is only serving `http` traffic